src/features/controls.ts

```diff
@@ -90,23 +90,24 @@ export const ControlsMixin = (ModelViewerElement:
             {type: String, attribute: 'camera-orbit', hasChanged: () => true})
         cameraOrbit: string = DEFAULT_CAMERA_ORBIT;
 
+        @property(
+            {type: String, attribute: 'camera-fov', hasChanged: () => true})
+        fieldOfView: string = DEFAULT_CAMERA_FOV;
+
         @property({type: Number, attribute: 'interaction-prompt-threshold'})
         interactionPromptThreshold: number =
             DEFAULT_INTERACTION_PROMPT_THRESHOLD;
 
         protected[$promptElement]: Element;
 
-        protected[$camera]: PerspectiveCamera;
-        protected[$fov]: number = DEFAULT_FOV;
-        protected[$idealCameraDistance] = 1.0;
-
         protected[$idleTime]: number = 0;
         protected[$userPromptedOnce]: boolean = false;
         protected[$waitingToPromptUser]: boolean = false;
         protected[$shouldPromptUserToInteract]: boolean = true;
 
         protected[$controls]: SmoothControls;
 
+        protected[$idealCameraDistance] = -1.0;
```

Comment:
Is `-1.0` a stand-in for "no value" here? If so, I would recommend using `null` instead, as it is more conventionally consistent with the rest of the library.